RELEASE_PROCESS.md

```diff
@@ -0,0 +1,80 @@
+# Release processes
+
+This document outlines the steps required in order to create a core protocol release.
+
+## Major/minor release process
+
+1. The default branch is: `develop`.
+1. Create a `release/vX.Y.Z` branch off the head of **`develop`**.
+1. Update all readme files, changelog, and set version strings as required:
+    - remove `Unreleased` from the changelog for the version to be released
+    - ensure that the readme is up-to-date for the version being released
+    - update the version number in `version/version.go`
+    - update the version number in `protos/sources/vega/api/v1/core.proto`
+    - update the version number in `protos/sources/vega/api/v1/corestate.proto`
+    - update the version number in `protos/sources/datanode/api/v2/trading_data.proto`
+    - update the version number in `protos/sources/blockexplorer/api/v1/blockexplorer.proto`
+    - run `make proto`
+    - run `git commit -asm "chore: release version vX.Y.Z`
+1. Create a pull request to merge `release/vX.Y.Z` into `master` and have it reviewed and merged
+1. Once the pull request has been merged, create a tag on the `master` branch
+1. The CI will see the tag and create all the release artefacts
```

Comment:
```suggestion
1. The CI will see the tag and create all the release artifacts
```

I know the spelling is interchangable, but GitHub uses `artifact` (https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts)